bears/general/LineCountBear.py

```diff
@@ -18,24 +19,58 @@ def _get_blank_line_count(self, file):
         return num_blank_lines
 
     def run(self, filename, file, max_lines_per_file: int,
-            exclude_blank_lines: bool=False):
+            ignore_files_regex: typed_list(str),
+            exclude_blank_lines: bool=False,
+            min_lines_per_file: int=1):
         """
-        Count the number of lines in a file and ensure that they are
-        smaller than a given size.
+        Count the number of lines in a file and ensure that they lie within
+        the range of given sizes. There are files for which lower limit isn't
+        required, but these files still shouldn't be rejected. Some of such
+        files when minimum is 1 are mentioned in ignore_files_regex
+        in the repository .coafile.
 
-        :param max_lines_per_file: Number of lines allowed per file.
-        :param exclude_blank_lines: "True" if blank lines are to be excluded.
+        :param max_lines_per_file:
+            Maximum number of lines allowed per file.
+        :param exclude_blank_lines:
+            ``True`` if blank lines are to be excluded.
+        :param min_lines_per_file:
+            Minimum number of lines required per file.
+        :param ignore_files_regex:
+            Files, for which minimum is not required.
```

Comment:
Please add another commit changing the docstring layout before for `max_lines_per_file` and `exclude_blank_lines`